### ISSUE TITLE
Unable to move zoomed image

### ISSUE THREAD

**Issue by shastah** [BestImageViewer/geeqie#499]
Whenever an image is zoomed to anything beyond "fit to screen" (ie. there are portions of the image file outside of the viewport), whether fullscreen or windowed, I can't drag and move the image around. Whenever I left-mouse-click on the image to start dragging, geeqie already loads next file and if I don't release left-mouse button, it drags that next picture.

I think it should wait for the mouse button to be released to load next image. And I'm pretty sure that was the behaviour around 1.1 (or even 1.2?).

My configure results are:
```
Package:
  Name:          geeqie
  Version:       035c3253

Architecture:
  UNIX:          yes
  Win32:         no (native: no)

Flags:
  Geeqie:        
  DEFS:          -DHAVE_CONFIG_H
  CPPFLAGS:      
  CFLAGS:        -O2 -D_FILE_OFFSET_BITS=64 -fPIC -Wall -Werror -Wno-error=deprecated-declarations
  CXXFLAGS:      -O2 -D_FILE_OFFSET_BITS=64 -fPIC -Wall
  Gtk:           -pthread -I/usr/include/gtk-3.0 -I/usr/include/at-spi2-atk/2.0 -I/usr/include/at-spi-2.0 -I/usr/include/dbus-1.0 -I/usr/lib64/dbus-1.0/include -I/usr/include/gtk-3.0 -I/usr/include/gio-unix-2.0/ -I/usr/include/cairo -I/usr/include/pango-1.0 -I/usr/include/atk-1.0 -I/usr/include/cairo -I/usr/include/pixman-1 -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/include/freetype2 -I/usr/include/harfbuzz -I/usr/include/libdrm -I/usr/include/libpng16 -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/libpng16 -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include
  Glib:          -pthread -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include
  Thread:        -lgthread-2.0 -pthread -lglib-2.0
  Others:	 -ljpeg -ltiff -llcms2 -lexiv2     

Localization:
  NLS support:   yes
  LINGUAS:        ar be bg ca cs da de el eo es et eu fi fr hu id it ja ko nb nl pl pt_BR ro ru sk sl sr@latin sr sv th tlh tr uk vi zh_CN zh_TW

Settings:
  Developer:     no
  Debug flags:   no
  Debug log:     yes
  Deprecated:    no

Support:
  LCMS:          yes
  Exiv2:	 yes
  Lirc:          no
  Clutter:       disabled
  Libchamplain:		disabled
  Libchamplain-gtk:	disabled
  Lua:	         no

Documentation:
  Doxygen:       /usr/bin/doxygen
  doc-tool:      /usr/bin/gnome-doc-tool

```

**Comment by xsdg** [comment]
Yeah, I've noticed this as well.

**Comment by caclark** [comment]
Could you check on the Preferences/Behaviour tab the setting of "Navigation by left or middle click on image"?
I think if you deselect it, the problem will disappear. If you want this setting to be "On", things will get a bit complex, unfortunately.

This was an option introduced last year. It defaults to the On setting. I think it would be better if the default was Off.

**Comment by shastah** [comment]
It does change behavior but still not to what I am used to from previous geeqie releases. Setting this to Off makes me unable to load new image using mouse click, which is something that worked very well in the past.

I consider this a regression.

**Comment by shastah** [comment]
On top of that, setting "Navigation by left or middle click on image" to "off" breaks `image.zoom_mode`.
Try this, with `image.zoom_mode` set to 1 in `geeqierc.xml`:
* start geeqie
* go to preferences and set "Navigation by left or middle click on image" to "off"
* navigate to an image that is bigger than your viewport
* press `z` to set 1:1 zoom
* load next image

You will see that the next image is also zoomed 1:1. When "Navigation by left or middle click on image" is set to "on", `zoom_mode` is preserved.

**Comment by caclark** [comment]
The second item is a bug separate from the "click" problem.
It is fixed in commit 4c3854b at geeqie.org.

**Comment by caclark** [comment]
Fixed in commit c7cb4da3c at geeqie.org
